.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "iotc-c-lib"]
```

Comment:
The individual versions pulled should be documented in either README.md or some other md file like at [this README](https://github.com/avnet-iotconnect/iotc-generic-c-sdk/blob/main/README.md#dependencies).

It is usually difficult to track which specific commit or tag is pulled of each module because .gitmodules doesn't support tags and is fixed to whatever is pulled in the repo. If you know of a better way to deal with this, I'm open to suggestions.